scripts/build

```diff
@@ -103,7 +103,6 @@ if [ ${ARCH} = riscv64 ]; then
     # These are needed as we are cross-compiling
     export CC=riscv64-linux-gnu-gcc
     export PKG_CONFIG_PATH=/usr/lib/riscv64-linux-gnu/pkgconfig/
-    export CGO_ENABLED=1
```

Comment:
Can you explain this change?

Reply:
The CGO_ENABLED flag was originally added by me. The [runc patch](https://github.com/opencontainers/runc/pull/3905/files) also includes Makefile changes that make it unnecessary, so I removed it.